composer.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "laravel/laravel",
+    "type": "project",
+    "description": "The Laravel Framework.",
+    "keywords": [
+        "framework",
+        "laravel"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.2",
+        "fideloper/proxy": "^4.0",
+        "laravel/framework": "^6.0",
+        "laravel/tinker": "^1.0"
+    },
+    "require-dev": {
+        "facade/ignition": "^1.4",
+        "fzaninotto/faker": "^1.4",
+        "mockery/mockery": "^1.0",
+        "nunomaduro/collision": "^3.0",
+        "phpunit/phpunit": "^8.0"
+    },
+    "config": {
+        "optimize-autoloader": true,
+        "preferred-install": "dist",
+        "sort-packages": true
+    },
+    "extra": {
+        "laravel": {
+            "dont-discover": []
+        }
+    },
+    "autoload": {
+        "psr-4": {
+            "App\\": "app/"
+        },
+        "classmap": [
+            "database/seeds",
+            "database/factories"
+        ]
```

Comment:
Thanks @NicholusMuwonge since you removed the folder for factories you should remove this line also `database/factories` otherwise it won't be possible to setup because whenever I try composer install I et the following error. In addition kindly add the screenshot homepage that we have currently on the PR Description. Other than that we are good to go.
![image](https://user-images.githubusercontent.com/22131999/66564317-64190c00-eb68-11e9-8a5c-0eb815004c1a.png)

Reply:
This will bring a lot of problems, in future. It has still been running on my side

Reply:
Yeah it works on your side because you already installed it before. To prove it clone the repo in the separate folder and try `composer install`